lib/galaxy/config/config_manage.py

```diff
@@ -219,21 +220,21 @@ class OptionValue(NamedTuple):
     "8080",
     ["galaxy.web.buildapp:app_factory"],
     "config/galaxy.yml",
-    str(GALAXY_CONFIG_SCHEMA_PATH),
+    GALAXY_CONFIG_SCHEMA_PATH,
```

Comment:
Hmm, isn't it better to pass around the pathlib instance ? I assume we eventually want to prefer pathlib over strings ?

Reply:
Oh, I see we do that :)